ISSUES

```diff
@@ -0,0 +1,12 @@
+- [ ] #1: Vendor mathjax
```

Comment:
delete this file and move issues into the repo (?)

Reply:
i was afraid of them getting lost in the github tho lol, but yeah